lib/modules/modules.go

```diff
@@ -92,9 +92,12 @@ type Features struct {
 	CustomTheme string
 
 	// IsTrialProduct is true if the cluster is in trial mode.
+	//
+	// TODO: Cannot use this field until cloud/salescenter sets this field
+	// when `func FetchFromCloud` and cloud/tenant.pb.go defines this field.
+	// see `e/lib/secreports/ func cloudLimits`, where it's currently the only
+	// place where we need to determine if product is trial.
```

Comment:
@smallinsky i'm thinking of just removing this altogether. it doesn't seem to be used,  and the place where we do need to check if it's trial, we do it [here](https://github.com/gravitational/teleport.e/pull/2829/files#diff-ca4645d13b462a0b3d84d3970e12fdf96b4c2c057de35f272cc327976fba2445R316), but we determine the value by fetching billing info from cloud.

remove it, and then re-introduce it when it's ready in the salescenter?